core/src/test/resources/HistoryServerExpectations/complete_stage_list_json_expectation.json

```diff
@@ -6,6 +6,7 @@
   "numCompleteTasks" : 8,
   "numFailedTasks" : 0,
   "executorRunTime" : 162,
+  "executorCpuTime" : 0,
```

Comment:
Oh, I though these were input to the history server, not "golden files" that it checks against... if that's the case, ignore my comment.